.github/workflows/build-test-deploy.yml

```diff
@@ -621,6 +621,35 @@ jobs:
           command: nix -- build .#devnet
           token: "${{ secrets.CACHIX_AUTH_TOKEN }}"
 
+  lint-test-frontend:
+    # Legacy because these should be nixified.
```

Comment:
imho may remove comment. we kind of know this. as soon as it will be non stable (because of vendor is not locked nor shared by nix) we would nixify.

Reply:
Just keeping that so FE knows why it is legacy.